<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Join</class>
 <widget class="QDialog" name="Join">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="fondo3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>900</width>
     <height>450</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextEdit" name="textEdit3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>670</width>
      <height>260</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(246, 245, 244);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>410</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Esperando que el jugador arranque el juego..</string>
    </property>
   </widget>
  </widget>
  <widget class="QSlider" name="slider3">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>770</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="iconoSonido3">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>770</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
